### PR TITLE
Add worker deployment for dev

### PR DESCRIPTION
**Pull request by pmac**
Also remove GCP deployment

### CODE REVIEW

File: frankfurt/nucleus-dev/web-deploy.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ metadata:
   labels:
     app: nucleus-dev
     type: web
-  name: nucleus-dev
+  name: nucleus-dev-web
```
Comment (pmac) [inline frankfurt/nucleus-dev/web-deploy.yaml:7]: @duallain any idea if this will cause issues? Wasn't sure about this one, but it's how basket is named.

Comment (duallain) [reply frankfurt/nucleus-dev/web-deploy.yaml:7]: as long as service/deployment match, it shouldn't be an issue.

Comment (bookshelfdave) [reply frankfurt/nucleus-dev/web-deploy.yaml:7]: here's the service: https://github.com/mozmeao/nucleus-config/blob/master/frankfurt/nucleus-dev/svc.yaml#L14-L16
the selector would have to change to match `nucleus-dev-web`

Comment (pmac) [reply frankfurt/nucleus-dev/web-deploy.yaml:7]: Would it? The `app:` labels match in both cases. But the `app:` in the service selector needs to match the `name:` from the deployment?

Comment (duallain) [reply frankfurt/nucleus-dev/web-deploy.yaml:7]: ahh, you're right pmac.  Name/selector being disjoint is perfectly fine, just wasn't always the convention before.

**Review by duallain — APPROVED** [review]

File: frankfurt/nucleus-dev/web-deploy.yaml (hunk 1 above)
Comment (pmac) [reply frankfurt/nucleus-dev/web-deploy.yaml:7]: Yeah. Basket uses `app: basket-dev` in the selector in the service so I think we're all good. Thanks y'all!

**Pull request merged**